build/hydrobuild.md

```diff
@@ -0,0 +1,335 @@
+---
+title: Hydrobuild
+description: Get started with Docker Hydrobuild
+sitemap: false
+---
+
+> **Restricted**
+>
+> Docker Hydrobuild is a early-access product.
+>
+> If you want to get involved in testing Hydrobuild, you can
+> [sign up for the early access program](https://www.docker.com/build-early-access-program/?utm_source=docs).
+{: .restricted }
+
+Hydrobuild is a service that lets you build your container images faster,
+locally and in CI. Builds run on cloud infrastructure optimally dimensioned for
+your workloads, no configuration required. The service uses a remote build
+cache, ensuring fast builds anywhere and for all team members.
+
+## How Hydrobuild works
+
+Using Hydrobuild is no different from running a regular build. You invoke a
+build the same way you normally would, using `docker build`. The difference is
+in where and how that build gets executed.
+
+By default when you invoke a `docker build` command, your build runs on a local
+instance of BuildKit, bundled with the Docker daemon. With Hydrobuild, you send
+the build request to a BuildKit instance running remotely, in the cloud.
+
+The remote builder executes the build steps, and sends the resulting build
+output to the destination that you specify. For example, back to your local
+Docker Engine image store, or to an image registry.
+
+Using Hydrobuild comes with several benefits out of the box:
```

Comment:
Not a dealbreaker: lines 34-41 sound a little 'marketing-y'. I wonder if it looks out of place with the rest of docs...at the same time I like the pace of this section, so just thought I'd flag it. Is Marketing doing anything for the EA release?

Reply:
I hear you and agree. This is very early days so no marketing content going out just yet. Thinking I leave this in here for now, we'll probably end up pruning it away later when it's not as new (maybe in a future freshness 😆)

Reply:
For sure, makes sense 👍